### ISSUE TITLE
How to resume training with another optimizer or lr ?

### ISSUE THREAD

**Issue by yuekaizhang** [espnet/espnet#2477]
Hi, I was wondering in espnet1 how to resume training from a snapshot or model. Currently, enc-init and dec-init could allow us to load a pretrained model. However, stuff like self.ctc could not be loaded. 
So, how to load an entire pretrained model? (Assume I would like to give a different learning rate comparing with snapshot)

**Comment by ShigekiKarita** [comment]
https://github.com/espnet/espnet/blob/59a48805aec1de33b8f528dab37a1da23cc8123d/espnet/bin/asr_train.py#L79-L85
How about `--resume`? It is nice to have a new helper script that customizes snapshots or models for `--resume` instead of `--{enc,dec}-init`.

**Comment by yuekaizhang** [comment]
Thanks. I check the function. The method would resume model and trainer at the same time. Perhaps I would comment the related code to resume model only rather than the trainer.
https://github.com/espnet/espnet/blob/59a48805aec1de33b8f528dab37a1da23cc8123d/espnet/asr/asr_utils.py#L745-L780
